src/include/basil.h

```diff
@@ -691,7 +792,13 @@ typedef struct basil_node {
 	basil_node_arch_t arch;
 	basil_node_state_t state;
 	basil_node_role_t role;
+	unsigned int numsockets;
+	unsigned int numdies;	/* dies = segments */
```

Comment:
Removed it, since we are not using it anywhere in the code. These are part of hyperthreading, and since this changeset doesn't support hyperthreading , we don't need these variables .